SPECS/cyrus-sasl-bootstrap/cyrus-sasl-bootstrap.spec

```diff
@@ -5,7 +5,7 @@
 Summary:        Cyrus Simple Authentication Service Layer (SASL) library
 Name:           %{_base_name}-bootstrap
 Version:        2.1.28
-Release:        3%{?dist}
+Release:        4%{?dist}
```

Comment:
I'd move the `cyrus-sasl` changes to a separate file, since they are fixing a separate bug.

Reply:
Separate PR?